communication/marketing-team/social-guidelines.md

```diff
@@ -24,19 +24,42 @@ Kubernetes Organization Members with access to account are:
 ## Themes/Topics
 
 - Contributor/SIG blog posts
-- Events
-- KEPs
+- Contributor events
+- KEPs*
+  - see [#KEPs] section for further guidance
 - Process changes
 - Release info, code freeze, et al
 - Accomplishments (reaching 100,000 GitHub stars for example)
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
+- Steering Committee Announcements
+
+## KEPs
+([Kubernetes Enhancement Proposals])
+
+### Ground Rules  
+- Social Media team can not proactively look for KEPs to amplify.
+- It's not ok to tweet about a KEP that is going to status: implementable until it's been merged.
+  - a good example is user feedback for features that are alpha
+- KEP metadata is found in the kubernetes/enhancements repo only and not on subsequent PRs.
+
+### Process  
+1. KEP requests must come from the author in the metadata; author will apply the area/contributor-comms label
+2. All tweets and content of tweet(s) must be approved explicitly on the KEP in GitHub by author/reviewer/approver listed in the metadata; the author is required and at least one party from the reviewer and/or approver category in the metadata
+3. If there is no meta data on the KEP, you can not amplify it on any social or communication channels
+4. To guide the discussion effectively, it's recommended to link to a discuss.kubernetes.io topic for the KEP unless asked to link to a GitHub issue by the Author.
 
 ### Frequently Asked Questions (FAQ)
 
-Q: Will the account follow other accounts?
+Q: Will the account follow other accounts?  
+
 A: TODO (Y/N). The account is designed for information dissemination, so we will focus on that by (following no one and focusing on sharing || following everyone to extend our reach).
 
-Q: Do we engage with hostile messages? 
+Q: Do we engage with hostile messages?  
```

Comment:
Do we need a "How do we advertise this process?" question or subheading? Are we going to make this known to the release team? or how is it otherwise "discoverable"

Reply:
@lachie83 we have an Issue template under way (PR to come), blog posts coming out, leads reaching out, and Paris setting up a time with the steering committee. More recommendations welcome (in Slack contribex channel, through issues, or on the meeting agenda).